docs/src/styles/docs/cardLink.scss

```diff
@@ -31,22 +20,89 @@
 
 .docs-cardLink-img {
   display: flex;
-  flex: 0 0 4rem;
+  flex: 0 0 3.5rem;
+  border-radius: var(--amplify-radii-small);
   svg {
     width: 100%;
     height: 100%;
     margin: auto;
   }
 }
 
-.docs-cardLink-title, .docs-cardLink-desc {
-  display: block;
-}
-
 .docs-cardLink-desc {
   color: var(--amplify-colors-neutral-80);
 }
 
 .docs-cardLink-title {
   font-weight: bold;
 }
+
+
+// Plain (default) variation
+.docs-cardLink--plain {
+  gap: var(--amplify-space-medium);
+  background: var(--amplify-colors-background-secondary);
+  box-shadow: var(--amplify-shadows-medium);
+  align-items: center;
+}
+
+
+
+// Branded and Home variations
```

Comment:
Branded and Home are nearly identical minus some little tweaks  that make it show up a bit better on the home page (i.e. shadow, no border)